RobotCode/src/main/java/frc/robot/Commands/ShooterCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Subsystems.ShooterSubsystem;
+
+public class ShooterCommand extends CommandBase
+{
+
+	private ShooterSubsystem shooter;
+
+	public ShooterCommand(ShooterSubsystem shoot)
+	{
+
+		shooter = shoot;
+		addRequirements(shooter);
+	}
+
+	public void execute()
+	{
+		shooter.setShooterMotorPower(1.0, "joystick said to shoot");
+	}
```

Comment:
This needs an isFinished() method to specify termination conditions, and an end() method to stop the command,

Should set motor power to 0 once finished

Reply:
isFinished() not needed because the command ends when button is released.